todo/src/components/TodoItem.js

```diff
@@ -1,17 +1,50 @@
 import React, { useState } from "react"
 
 function TodoItem(props) {
-	const [checked, setChecked] = useState(false)
 	return (
-		<div>
+		<div
+			style={{
+				display: "flex",
+				border: `1px solid ${props.darkModeEnabled ? "#f1f1f1" : "#333333"}`,
+				marginBottom: "1.5vh",
+				borderRadius: "5px",
+			}}
+		>
 			<label>
 				<input
+					style={{
+						borderRadius: "5px",
+					}}
 					type="checkbox"
-					defaultChecked={checked}
-					onChange={() => setChecked(!checked)}
+					defaultChecked={props.cb}
+					onChange={() => {
+						props.onChange(props.id)
+					}}
 				/>
 			</label>
-			{props.index + 1 + " " + props.content}
+			<label
+				style={{
+					textDecoration: props.cb && "line-through",
+					opacity: props.cb && "0.5",
+				}}
+			>
+				{props.index + 1 + ". " + props.content}
+			</label>
+
+			<button
+				style={{
+					marginLeft: "auto",
+					backgroundColor: props.darkModeEnabled ? "#333333" : "#f1f1f1",
+					color: props.darkModeEnabled ? "#f1f1f1" : "#333333",
+					border: `1px solid ${props.darkModeEnabled ? "#f1f1f1" : "#333333"}`,
+					borderRadius: "5px",
+				}}
+				onClick={() => {
+					props.onClick(props.id)
+				}}
+			>
+				x

```

Comment:
Usually an icon would be better but in this case, just stick to the requested design.